src/java-runtime/java-runtime.targets

```diff
@@ -6,10 +6,7 @@
     <OutputDex>$(OutputPath)java_runtime.dex</OutputDex>
     <IntermediateRuntimeOutputPath>$(IntermediateOutputPath)release</IntermediateRuntimeOutputPath>
     <IntermediateRuntimeClassesTxt>$(IntermediateOutputPath)release.txt</IntermediateRuntimeClassesTxt>
-    <RemoveItems>java\mono\android\debug\MultiDexLoader.java</RemoveItems>
-    <RemoveItems>java\mono\android\MonkeyPatcher.java</RemoveItems>
-    <RemoveItems>java\mono\android\ResourcePatcher.java</RemoveItems>
-    <RemoveItems>java\mono\android\Seppuku.java</RemoveItems>
+    <RemoveItems>java\mono\android\debug\MultiDexLoader.java;java\mono\android\MonkeyPatcher.java;java\mono\android\ResourcePatcher.java;java\mono\android\Seppuku.java</RemoveItems>
```

Comment:
Why "merge" these lines?  I find the easier version easier to read.

Reply:
> The RemoveItems property of _RuntimeOutput was being
treated like an ItemGroup when it is infact a property.
So we need to semi-colon seperate the items to make sure
they are ALL removed rather then it just being the last one
listed.

So Metadata are Properties not ItemGroups.. so having them all on separate lines results in the metadata having only the LAST item. Cos each line overwrote the other ones. So we need to have them all in one line and `;` separated so its compatible with the `Remove` attribute on ItemGroups.